app/controllers/avo/associations_controller.rb

```diff
@@ -65,13 +65,18 @@ def create
 
       respond_to do |format|
         if @model.save
-          format.html { redirect_back fallback_location: resource_path(model: @model, resource: @resource), notice: t("avo.attachment_class_attached", attachment_class: @related_resource.name) }
+          format.html { redirect_back fallback_location: resource_default_view_after_path, notice: t("avo.attachment_class_attached", attachment_class: @related_resource.name) }
         else
           format.html { render :new }
         end
       end
     end
 
+    def resource_default_view_after_path
```

Comment:
@adrianthedev here and in the `base_controller.rb`i had to change this method name to prevent helper'se method overriding, i'm open to name suggestions, maybe `resource_default_view_response_path` ??